docs/manual/src/udl/callback_interfaces.md

```diff
@@ -47,7 +47,7 @@ which may use it.
 In UDL:
 
 ```webidl
-object Authenticator {
+interface Authenticator {
```

Comment:
I think this should actually be `callback interface Authenticator {`

Reply:
```suggestion
callback interface Authenticator {
```

Reply:
Are you sure? As I read it, the `Authenticator` is an interface that _uses_ the `callback interface Keychain`.

Reply:
Ahh yes, my mistake.  Merging now.